testutils/test_server.go

```diff
@@ -78,7 +78,11 @@ func NewTestServer(t testing.TB, opts *ChannelOpts) *TestServer {
 
 	ts.NewServer(opts)
 	if opts.IncludeRelay {
-		ts.addRelay()
+		var relayLogOpts LogVerification
+		if opts != nil {
```

Comment:
Can `opts` be nil here? Seems like we'd already have paniced on L80. Maybe `NewTestServer` should handle nil options at the top.